.github/workflows/update_files_for_release.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow is triggered by manual inputs.
+
+name: Update files for the new release
+
+on:
+  workflow_dispatch:
+    inputs:
+      OLD_VERSION:
```

Comment:
The release after a quarterly release (version ending in .3, .6, .9, .12) wouldn't have an "old" version. The field is marked as required. Just want to make sure that such scenarios would work fine?

Reply:
We won't always have "old" version. For example, after we release 23.0.0.12, we'll only add 24.0.0.1

Reply:
As long as the folders for the quarterly releases are not removed before they are supposed to be, we are okay. It seems like that would be the case from your description.